python/grass/jupyter/utils.py

```diff
@@ -69,9 +71,16 @@ def reproject_region(region, from_proj, to_proj):
         raise RuntimeError(
             _("Encountered error while running m.proj: {}").format(stderr)
         )
-    enws = gs.decode(proj_output).split(os.linesep)
-    elon, nlat, unused = enws[0].split(" ")
-    wlon, slat, unused = enws[1].split(" ")
+    enws = gs.decode(proj_output).splitlines()
+    # get the largest bbox
+    n = []
+    e = []
```

Comment:
Pylint fails here with current settings. You can rename it or add enws to `good-names` in `.pylintrc`. enws is likely used elsewhere where Pylint is not used. The following is already there:

```
           i,
           j,
           k,
           x,
           y,
           z,
```

We may end up with the whole alphabet there. `abc` for `c = a + b`, `t` for time. I'm not sure.

Reply:
These two are anyway longitudes and latitudes or something else in plural (for lists).